libs/entry-components/dialog/dialogs/entry-dialog.component.html

```diff
@@ -1,33 +1,19 @@
-<div class="entry-dialog dialog-form">
-    <div class="dialog-header">
-        <h1 class="title">{{ title }}</h1>
+<h1 mat-dialog-title>{{ title }}
+    <button mat-icon-button mat-dialog-close aria-label="Close dialog" *ngIf="!hideClose">
+        <mat-icon>close</mat-icon>
+      </button>
+</h1>
+<div mat-dialog-content>
+    <ng-content></ng-content>
+</div>
```

Comment:
How does this new alignment work? This [align] thingy comes from angular material?